_posts/2020-06-25-changelog-125.markdown

```diff
@@ -0,0 +1,176 @@
+---
+layout: post
+title: "IntelliJ Rust Changelog #125"
+date: 2020-06-25 19:26:11 +0300
+---
+
+
+## New Features
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5535 -->
+* Support debugging with MSVC toolchain in CLion 2020.2
```

Comment:
Let's also mention https://github.com/intellij-rust/intellij-rust/issues/5632 as the current status of debugger for msvc toolchain